ShoppingCart/ShoppingCart.java

```diff
@@ -0,0 +1,122 @@
+import java.util.*;
+
+public class ShoppingCart 
+{
+	int total;
```

Comment:
Need to start using access modifiers. All the variables of a class should be private, and should be accessible through functions (getters/setters) that can be public.